src/builtins.c

```diff
@@ -436,7 +436,8 @@ JL_CALLABLE(jl_f_apply)
             jl_value_t *argarr = jl_apply(jl_append_any_func, &args[1], nargs-1);
             assert(jl_typeis(argarr, jl_array_any_type));
             JL_GC_PUSH1(&argarr);
-            jl_value_t *result = jl_apply(f, jl_cell_data(argarr), jl_array_len(argarr));
+            jl_value_t *result = jl_apply(f, jl_cell_data(argarr),
```

Comment:
I guess you prefer an 80-column limit? Anyway usually better to avoid unrelated formatting changes like this.